frontend/app-development/layout/AppBar/AppBar.tsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { AppBar as MuiAppBar, Grid, Toolbar } from '@mui/material';
 import classNames from 'classnames';
 import { Link, useParams } from 'react-router-dom';
-import { altinnImgLogoHeaderUrl } from 'app-shared/cdn-paths';
 import { getTopBarMenu } from './appBarConfig';
 import { ProfileMenu } from 'app-shared/navigation/main-header/profileMenu';
 import { VersionControlHeader } from '../version-control/VersionControlHeader';
 import { getRepositoryType } from 'app-shared/utils/repository';
 import classes from './AppBar.module.css';
 import { useMediaQuery } from '../../common/hooks';
 import { useTranslation } from 'react-i18next';
-import { Settings } from '@navikt/ds-icons';
+import AltinnStudioLogo from 'app-shared/navigation/main-header/AltinnStudioLogo';
+import { ThreeDotsMenu } from 'app-shared/navigation/main-header/ThreeDotsMenu';
+import { AiOutlineBranches } from 'react-icons/ai';
```

Comment:
Vi bruker `@navikt/aksel-icons` til ikoner. Fint om vi ikke drar inn for mange andre avhengigheter. Se om du finner et ikon som passer her: https://aksel.nav.no/ikoner
Du bør også merge inn nyeste versjon av master, så får du med oppdateringen hvor vi har byttet ut den gamle ikonpakken til Nav.

Reply:
har erstattet den